flexport/latitude
# Resolve AF-524 SearchableMultiselectFilter input handles pasting

This PR resolves issue https://flexport.atlassian.net/browse/AF-524, fixing a bug where text couldn't be pasted into the SearchableMultiselectFilter's search input.

This PR adds a test that fails before and passes after with the addition of this bug fix.

## Manual Testing Plan
Check out this branch and go to the SearchableMultiselectFilter story in storybook
Replicate the steps in the description of https://flexport.atlassian.net/secure/RapidBoard.jspa?rapidView=384&view=detail&selectedIssue=AF-524
Verify that issue is resolved.
